test/tests/rackhd20/test_rackhd20_api_notifications.py

```diff
@@ -0,0 +1,265 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+Author(s):
+'''
+
+from time import sleep
+import Queue
+import flogging
+import logging
+import pika
+import unittest
+import threading
+import fit_common
+from nose.plugins.attrib import attr
+logs = flogging.get_loggers()
+amqp_queue = Queue.Queue(maxsize=0)
+from pymongo import MongoClient
+from bson.objectid import ObjectId
+
+IP = None
+
+
+class AmqpWorker(threading.Thread):
+
+    def __init__(self, exchange_name, topic_routing_key, external_callback, timeout=10):
+        threading.Thread.__init__(self)
+        pika_logger = logging.getLogger('pika')
+        if fit_common.VERBOSITY >= 8:
+            pika_logger.setLevel(logging.DEBUG)
+        elif fit_common.VERBOSITY >= 4:
+            pika_logger.setLevel(logging.WARNING)
+        else:
+            pika_logger.setLevel(logging.ERROR)
+        if fit_common.API_PORT == "9090":
+            amqp_port = fit_common.fitports()['amqp-vagrant']
```

Comment:
heads up for this -  changes for 'amqp-vagrant', and 'amqp' ports variables are coming PR 715, so ampq_port will need to change to this.
 amqp_port = fit_common.fitports()['amqp_ssl']
We found the tests shouldn't need to check if api-port is 9090 inside the script to get the correct values it needs, so config files being updated to work properly.  9090 is just how our vagrant is set up.